src/mainboard/sunxi/nezha/Makefile

```diff
@@ -0,0 +1,28 @@
+OREBOOT=$(abspath $(CURDIR)/../../../../)
+# RV64GCV, ackshully
+TARGET     = riscv64gc-unknown-none-elf
+QEMU       ?= qemu-system-riscv64
+QEMU_FLAGS += -m 1g -machine sifive_u,start-in-flash=true -nographic -device loader,addr=0x20000000,file=${IMAGE} -bios none -smp 4
+XFEL       ?= xfel
+IMAGE      ?= target/$(TARGET)/release/image.bin
+# DRAM space starts at 0x40000000, see manual p34
+MEMADDR    ?= 0x40000000
```

Comment:
Do you really want the ?= operator here? is it really possible to change MEMADDR?

Reply:
Yea I thought about it. It's temporary anyway until we can boot from flash (and actually flash). One idea was to extend the Makefile with a little `make load` where we could load something else somewhere for testing bits and pieces gradually. I'll think about it.